bucket/jdtls.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "1.10.0-202204081744",
+    "description": "Java language server",
+    "homepage": "https://projects.eclipse.org/projects/eclipse.jdt.ls",
+    "license": "EPL v2.0",
```

```suggestion
    "license": "EPL-2.0",
```
See [SPDX license identifiers](https://spdx.org/licenses/).

---

Fixed!